### ISSUE TITLE
Multiple entries of the same organization is allowed 

### ISSUE THREAD

**Issue by AnnCherianA** [Bridgeconn/vachan-api#104]
Issue Description : The same organization can be created multiple times against the same email id resulting in duplicate entries. 
Steps:
Login to AgMT as a translator
Click request admin access
Enter the required details such as organization,  email and phone number 
Observation : An alert saying organization request sent is displayed

Now click request admin permission again and enter the same organization, same email and phone number.
Observation : It is possible tor create the same organization under the same email login and phone number, resulting in duplication 
Expected Result: Once an organization is created, creating the same organization should be disabled for the same login

**Comment by AnnCherianA** [comment]
![AgMT vachan-api-Duplicate entries of organization are created](https://user-images.githubusercontent.com/74647279/102882739-dc6da380-4474-11eb-95cb-ad8cb2003dbc.jpg)

**Comment by kavitharaju** [comment]
The 3 BCSs were not exactly the same
```
organisation_name | char_length
------------------------- +-------------
 BCS|           4
 BCS                       |           3
  BCS                      |           4
```

The stray spaces caused the confusion here. This issue will not occur now after this [fix](#103) where we remove any trailing or preceeding spaces from organization names.
